symfony/plugins/orangehrmAdminPlugin/Api/Controller/EmploymentStatusApiController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/*
```

Comment:
Can you update the copyright header in php files, copy from symfony/plugins/orangehrmAdminPlugin/Api/UserAPI.php